src/pages/index.tsx

```diff
@@ -19,7 +19,7 @@ export function getStaticProps() {
     .map(filePath => {
       const prayerData = getPrayerDataFromFilename({ filePath });
       const source = fs.readFileSync(path.join(PRAYERS_PATH, filePath));
-      const { data, content } = matter(source);
+      const { content, data } = matter(source);
```

Comment:
do we have an unit test for this change?

Reply:
in destructuring objects, it doesn't matter

try
```
const { cat, dog } = { cat: "Olek", dog: "Reksio" };
```
and 
```
const { dog, cat } = { cat: "Olek", dog: "Reksio" };
```
Both will give you the same values for the cat.

Reply:
I used this change to trigger a new deploy, will undo this soon